middlewares/runtime-errors.js

```diff
@@ -52,6 +52,16 @@ const openApiError = (err, req, res, next) => {
     `${error.path} must be one of ['${error.params.allowedValues.join("', '")}']`
   );
 
+  /**
+   * Generate error message for incorrect path parameter formats
+   *
+   * @param {object} error The error object
+   * @returns {string} The error message
+   */
+  const invalidPatternMessage = error => (
+    `Error in path: '${error.path}', location: '${error.location}', message: '${error.message}'`
```

Comment:
Can you make this message easier to read? For example:

```js
`'${error.path}' in path ${error.message}`
```

It'll generate the message to be: `'id' in path should match pattern \"^\\d+$\"`. Also, since this function will only be used once, I think you don't need to create a funciton for it separatly. You can directly replace `invalidPatternMessage(invalidPatternError)` at line 83 with this template string.